app/models/event.rb

```diff
@@ -3,10 +3,26 @@ class Event < ActiveRecord::Base
   belongs_to :category
   has_many :ticket_types
 
-  validates_presence_of :extended_html_description, :venue, :category, :starts_at
-  validates_uniqueness_of :name, uniqueness: {scope: [:venue, :starts_at]}
+  validates :name, :extended_html_description, :venue, :category, :starts_at, presence: true
+  validates :name, uniqueness: { scope: [:venue_id, :starts_at] }
 
-  def starts_at_to_string
+  def starts_at_to_s
     starts_at.strftime(FULL_DATE_FORMAT)
   end
+
+  def starts_at_to_month
+    starts_at.strftime(FULL_MONTH_FORMAT)
+  end
+
+  def starts_at_to_day
+    starts_at.strftime(DAY_FORMAT)
+  end
+
+  def region_name
+    venue.region.name
+  end
+
+  def min_price
+    ticket_types.map { |type| type.price  }.min
```

Comment:
This is quite bad. It would load all associated TicketType objects, then pick up prices for each object to get an array, and only then choose the smallest one. Find a way to optimize this at Database level.